pkg/cmd/server/kubernetes/node_config.go

```diff
@@ -190,6 +185,23 @@ func BuildKubernetesNodeConfig(options configapi.NodeConfig, enableProxy, enable
 		return nil, err
 	}
 
+	// Initialize SDN before building kubelet config so it can modify options
```

Comment:
ok, so the rearrangement from how it was before is not actually needed, it's just aesthetic? I guess the old way keeps all of the `server` initialization code together, while the new way keeps all the SDN code together...